Editor/UI/Windows/StepWindow.cs

```diff
@@ -89,6 +89,11 @@ public void SetStep(IStep newStep)
             step = newStep;
         }
 
+        public IStep GetStep()
```

Comment:
Missing docs

Reply:
This is a member of an internal class, and I do not know how could you misinterpret what does this method do.